doc/md/fundamentals/2-types/12-advanced-types.md

```diff
@@ -104,12 +105,12 @@ In the example above, `T <: { name : Text }` requires that any type used for `T`
 
 Type bounds are not limited to records. For example, it is possible to constrain a generic type to be a subtype of a basic type.
 
-```motoko no-repl
+```motoko
 func addIfInt<T <: Int>(x : T, y : T) : Int {
   return x + y;
 };
 
```

Comment:
```suggestion
```motoko
func max<T <: Int>(x : T, y : T) : T {
  if (x <= y) y else x
};
```
Is a better example because it now returns a `Nat` when applied to `Nat`, and `Int` when applied to `Int` (the return type depends on T). Otherwise you could just write addIfInt(x : Int; y:Int) : Int which is simpler and has the same utility.

The record example could also be better, for similar reasons.
```